File: fibonacci.java
class fibonacci {
    public static void main(String[] args)
    {
        int n = 40;
	int sum;

	final long startTime = System.currentTimeMillis();
        sum = fib(n);
	final long endTime = System.currentTimeMillis();
        
        System.out.println("fib(" + n + ") = " + sum);
	System.out.println("time    = " + (endTime - startTime) + " milliseconds");
    }
    
    static int fib(int n)
    {
        if (n < 3)
            return 1;
        else
            return fib(n-1) + fib(n-2);
    }
}

File: fibonacci.py
import time

# Bypass Python's internal limit on number of recursive function calls.
#from functools import lru_cache

#@lru_cache(maxsize=None)
def fibr(n):
    if n < 3:
        return 1
    else:
        return fibr(n-1) + fibr(n-2)

n = 40
sum = int()

start = time.time() * 1000
sum = fibr(n)
end = time.time() * 1000
delta = int(end - start)
#delta = time.time() * 1000 - start

# Print as required
print(f"fib({n}) = {sum}")
print(f"time    = {delta} milliseconds")
